projects/igniteui-angular/src/lib/core/utils.ts

```diff
@@ -788,3 +788,9 @@ export const formatDate = (value: string | number | Date, format: string, locale
 };
 
 export const formatCurrency = new CurrencyPipe(undefined).transform;
+
+/** Converts pixel values to their rem counterparts for a base value */
+export const rem = (value: number | string) => {
+    const base = parseFloat(getComputedStyle(document.documentElement).getPropertyValue('font-size'))
```

Comment:
Hey there! my two cents for this, I'm a bit concerned about this function, as it is doing calculations and interacting with dom properties directly from the template rendering.

My concern is that doing that kind of interactions can seriously impact performance, a better approach would be to do calculation of `base` on `ngOnInit` (only once) and then updating that variable value on certain events (like resizing).